content/projects/intro-to-database/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Introduction to database
+
+ready: True
+
+## Task
+
+- create a database and name it “mystore”
+- Create tables the following tables :customers, product items, suppliers, orders.
+- Add properties to table (be creative here)
+- Link tables  (relationships)
+- Write a query to populate each table with at least 5 items
+- Write a query to check the amount of product item left
+- write a query to check the amount of orders
+- All commands and queries to written within a file named my store.sql
+- Setup docker on your computer
```

Comment:
Maybe link to the docker setup page of the repo: https://umuzi-org.github.io/tech-department/topics/intro-to-docker/